cmd/argocd/commands/admin/redis_initial_password.go

```diff
@@ -23,8 +23,8 @@ import (
 )
 
 const (
-	defaulRedisInitialPasswordSecretName = "argocd-redis"
-	defaultResisInitialPasswordKey       = "auth"
+	DefaultRedisInitialPasswordSecretName = "argocd-redis"
+	DefaultRedisInitialPasswordKey        = "auth"
```

Comment:
This can probably be moved to common/common.go to help avoid import cycled.

Reply:
Thanks!  I shifted them in there and consolidated the secret fetching logic as well